core/csvcollector.class.inc.php

```diff
@@ -306,6 +295,20 @@ protected function Configure($aCsvHeaderColumns)
 		}
 	}
 
+	/**
+	 * @inheritdoc
+	 */
+	protected function CheckColumns($aSynchroColumns, $aColumnsToIgnore, $sSource)
+	{
+		Utils::Log(LOG_DEBUG, "[".get_class($this)."] Columns [".var_export($this->aSynchroColumns, true)."]");
+		foreach ($this->aFields as $sSynchroColumn => $aDefs) {
+			if (array_key_exists($sSynchroColumn, $this->aSynchroFieldsToDefaultValues) || in_array($sSynchroColumn, $this->aIgnoredSynchroFields)) {
+				$aColumnsToIgnore[] = $sSynchroColumn;
+			}
+		}
+
+		parent::CheckColumns($this->aSynchroColumns, $aColumnsToIgnore, 'csv file');
```

Comment:
```suggestion
		parent::CheckColumns($this->aSynchroColumns, $aColumnsToIgnore, $sSource);
```

Shouldn't this rather use the given parameter?